tests/integration/redis/test_instance_basic.php

```diff
@@ -23,6 +23,9 @@
 ok - get key
 ok - delete key
 ok - delete missing key
+ok - mset key
+ok - mget key
+ok - delete key
```

Comment:
I realized that I forgot to add these few lines in my last commit.
As far as I understand this, it should be mandatory to compare expected/actual results… so how/why did the tests pass without these lines? 🤔

Reply:
Hi, @b-viguier!

Looks like the tests didn't run yet because the workflow needs to be approved.  I'll approve them now and watch them run :)